src/util/ScreenSaver/ScreenSaver.h

```diff
@@ -0,0 +1,126 @@
+#include "Cemu/Logging/CemuLogging.h"
+
+#if BOOST_OS_MACOS
+#include <IOKit/pwr_mgt/IOPMLib.h>
+#endif
+
+#if BOOST_OS_LINUX
+#include <spawn.h>
```

Comment:
It seems that including [`spawn.h`](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/spawn.h.html) is unneeded.